dynamodb-streams-lambda-filters-tf/README.md

```diff
@@ -0,0 +1,80 @@
+# DynamoDB Stream to AWS Lambda with Filters 
+
+This pattern demonstrates how to lambda functions that can recieve DynamoDB Stream with filters. The IaC has been definied in Terraform. 
+
+Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [Terraform Installed](https://developer.hashicorp.com/terraform/downloads)
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory and the terraform directory:
+    ```
+    cd dynamodb-streams-lambda-filters-tf/tf
+    ```
+1. From the command line, run the following command to initializes terraform:
+    ```
+    terraform init
+    ```
+1. From the command line, run the following command to cerate terraform execution plan:
+    ```
+    terraform plan
+    ```
+1. From the command line, run the following command to excute the deployment:
+    ```
+    terraform apply
+    ```
+
+    > ** if choose skip the confirmation prompt, use -auto-approve
+
+## How it works
+
+DynamoDB Streams trigger Lambda Function based on filter. The stack has one DynamoDB Table with 4 Lambda with selective filters,
```

Comment:
A DynamoDB stream triggers different Lambda functions based on a filter. The stack has one DynamoDB table with 4 functions with selective filters: